src/tags/object/Paragraphs/Phrases.js

```diff
@@ -4,13 +4,19 @@ import { Button } from 'antd';
 import { PauseCircleOutlined, PlayCircleOutlined } from '@ant-design/icons';
 import styles from './Paragraphs.module.scss';
 
-export const Phrases = observer(({ item }) => {
+const sortPhrases = (a, b) => {
+  if (!a.start) return 1;
+  if (!b.start) return -1;
+  return a.start - b.start;
+};
+
+export const Phrases = observer(({ item, contextScroll }) => {
   const cls = item.layoutClasses;
   const withAudio = !!item.audio;
 
   if (!item._value) return null;
-
-  const val = item._value.map((v, idx) => {
+  const list = contextScroll ? item._value.sort(sortPhrases) : item._value;
```

Comment:
We will need this to be sorted within the model level so anywhere that references this data it is the same.